internal/protobuf/helpers.go

```diff
@@ -101,3 +103,15 @@ func AnyMessageTypeOf(msg proto.Message) string {
 	a := MustMarshalAny(msg)
 	return a.TypeUrl
 }
+
+/// MustMarshalJSON marshals msg to indented JSON.
+func MustMarshalJSON(msg proto.Message) string {
```

Comment:
I don't see this being used anywhere, can remove if that's the case.

Reply:
I ended up having to rewrite it in various ad-hoc debugging, so even though it's not used I'd like to keep it for future debug logging :)